### ISSUE TITLE
Crash (or undefined behavior) on calling plugin actions/commands from scripts

### ISSUE THREAD

**Issue by YannickJadoul** [praat/praat#1583]
After loading a plugin, or after basically adding anything with "Add action command", the action cannot be run from a script (not even in the built-in script editor in Praat).

The reason is that `praat_doAction` (in `sys/praat_actions.cpp`) completely ignores the `Praat_Command`'s `script`, and passes a `nullptr` to `callback` (i.e., `DO_RunTheScriptFromAnyAddedMenuCommand`).

I've investigated, but fixing it is far from easy (since `DO_RunTheScriptFromAnyAddedMenuCommand` won't even look at the call's arguments that are passed), so I won't submit a PR myself.

**Comment by PaulBoersma** [comment]
This is intended behaviour. To run a script from another script, call it by using `runScript`. For instance, to call a plugin action from outside the plugin itself, you can do:

    pluginFolder$ = preferencesDirectory$ + "/plugin_yannick/"
    runScript: pluginFolder$ + "doSomething.praat", firstArgument, secondArgument$

**Comment by YannickJadoul** [comment]
OK, I didn't know that this was intended behavior.

Thank you for the workaround. I did know the plugins can be run by running the script itself, but I didn't realize by reading the docs that the new actions/menu commands are not meant to be used in scripts. I also got puzzled by the error message looking something like "Can never allocate -9,233,372,036,854,775,808 bytes" (which seem to indicate undefined behavior happening).

As this is intended behavior, I will close this issue, then.
